### PR TITLE
Disable test for coverage cuda12

### PR DESCRIPTION
**Pull request by tianshuo78520a**
### PR types

Others
### PR changes

Others
### Description

CUDA12 单测失败，临时禁用单测
Pcard-67012

### CODE REVIEW

File: test/auto_parallel/CMakeLists.txt [hunk 1]
```diff
@@ -242,3 +242,7 @@ if(WITH_DISTRIBUTE AND WITH_GPU)
   # End of unittests WITH single card WITHOUT timeout
 
 endif()
+
+if(WITH_COVERAGE)
```
Comment (XieYunshen) [inline test/auto_parallel/CMakeLists.txt]: 应该把在这个之前加一下判断条件
image

File: test/collective/CMakeLists.txt [hunk 2]
```diff
@@ -479,3 +479,7 @@ if((WITH_ROCM OR WITH_GPU) AND (LINUX))
 endif()
 add_subdirectory(fleet)
 add_subdirectory(multinode)
+
+if(WITH_COVERAGE)
+  list(REMOVE_ITEM TEST_OPS test_collective_reduce_scatter_api)
```
Comment (XieYunshen) [inline test/collective/CMakeLists.txt]: 判断条件里面加一个NOT WITH_COVERAGE
image

Comment (tianshuo78520a) [reply test/collective/CMakeLists.txt]: done

File: test/auto_parallel/CMakeLists.txt (hunk 1 above)
Comment (tianshuo78520a) [reply test/auto_parallel/CMakeLists.txt]: done

**Review by XieYunshen — DISMISSED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]
LGTM

**Pull request merged**